include/JoystickHandler.h

```diff
@@ -42,6 +42,7 @@ namespace hri_safety_sense {
 
 		  ros::NodeHandle 		rosNode;
 		  ros::Publisher 		rawLeftPub, rawRightPub;
+		  bool 					reconfigureJoy{false};
```

Comment:
```suggestion
		  bool 				useArrowsAsAxes{ false };
```

It's helpful to have a more descriptive name for the parameters. One that clarifies exactly what that parameter affects.